db-builder/src/main/java/com/yahoo/covid19/database/models/Places.java

```diff
@@ -53,23 +53,24 @@
     private final String longitude;
     private final String latitude;
     private final String population;
-    private final String stateId;
-    private final String countryId;
+    private final List<String> parentId;
 
     @JsonAdapter(TableNameAdapter.class)
     private final JoinTableNames type;
 
     private transient ErrorCodes errorCode = OK;
+    private transient Boolean isValid = null;
```

Comment:
This see like a hack - why do we need state for this? We already have an isValid method.